src/webui/modules/home/index.js

```diff
@@ -118,6 +96,24 @@ export default class Home extends React.Component {
   }
 
   renderPackageList() {
-    return <PackageList help={this.state.fistTime} packages={this.state.packages} />;
+    let packages = [];
+    packages = this.state.query ? this.getFilteredPackages(this.state.packages, this.state.query) : this.state.packages;
+
+    return <PackageList help={this.state.fistTime} packages={packages} />;
+  }
+
+  /**
+   * Function tp return an array of Package that contains
+   * the query in 'name' & 'description' field
+   * @param {*} packages
+   * @param {*} query
+   */
+  getFilteredPackages(packages = [], query) {
+    return packages.filter((obj) => {
+      return (
+        obj.name.includes(query) ||
```

Comment:
Please use `toLowercase` on name and description before chaining includes a method, in this way we can get more expected results. For example with current code if we have `jQuery` it will not be retrieved with query `jquery`.

See JS Bin snippet here [https://jsbin.com/seceboxuqi/edit?js,console](https://jsbin.com/seceboxuqi/edit?js,consoles)

Reply:
also when possible use lodash method which provide safe calls.